packages/artificial-tweet/src/direflow-components/artificial-tweet/components/artificial-tweet.tsx

```diff
@@ -0,0 +1,65 @@
+import {
+  Box,
+  Button,
+  SelectChangeEvent,
+  Stack,
+  Typography,
+} from '@mui/material';
+import React, { useState } from 'react';
+import { GeneratedTweet, Party } from '../../../types';
+import { useFetch } from '../hooks/use-fetch';
+import { MultiInputField } from './fields/multi-input-field';
+import { SelectField } from './fields/select-field';
+import { Tweet } from './tweet';
+
+export const ArtificialTweet: React.FunctionComponent = () => {
+  const { apiRequest } = useFetch();
+  const [party, setParty] = useState<Party>(Party.parteilos);
+  const [keywords, setKeywords] = useState<string[]>([]);
```

Comment:
This is rlly cool. But this is only useful when the backend supports multiple keywords. Is this the case? From the last meetings @soosaasees said that the backend would receive a prompt and not keyword(s)